src/test/java/ca/bc/gov/backendstartapi/dto/UserDtoTest.java

```diff
@@ -0,0 +1,57 @@
+package ca.bc.gov.backendstartapi.dto;
```

Comment:
This may be a bit much. Do we need an unit test for a DTO?

We have no logic in our getters and setters, so I don't see what's to be tested here, besides their existence (which would be done as part of other tests anyway).

We're rolling our own hash function, so testing `hashCode()` seems like a non-issue.

The rest of the methods would be verified in an integration test where the DTO were used, since we'd need to check if what we received from an operation is or isn't empty and is equal to what we expected.

And we'd have to update all of them if we added a new field.

I think this is an interesting take on testing: https://www.youtube.com/watch?v=EZ05e7EMOLM

Reply:
Yeah, we actually need, based on the minimum test coverage rate. Unless we discard dto objects.